src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImage.groovy

```diff
@@ -15,10 +15,21 @@
  */
 package com.bmuschko.gradle.docker.tasks.image
 
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
 class DockerRemoveImage extends DockerExistingImage {
+
+    @Input
+    @Optional
+    Boolean force
+
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Removing image with ID '${getImageId()}'."
-        dockerClient.removeImageCmd(getImageId()).exec()
+        def removeImageCmd = dockerClient.removeImageCmd(getImageId())
```

Comment:
hmmm ... @orzeh if we one-line this as you noted I believe we would have to do something like this: `dockerClient.removeImageCmd(getImageId()).withForce(getForce() ? getForce().booleanValue() : false).exec()` 

Otherwise it's possible the user could set this to `null` which is not going to make `docker-java` happy. I don't think groovy will attempt to massage the `null` into `false` for java libraries. Assuming I'm right it may look cleaner to do things the previous way.